contrib/libzookeeper/src/mt_adaptor.c

```diff
@@ -441,6 +441,8 @@ void *do_io(void *v)
         // if it is_unrecoverable()
         if(is_unrecoverable(zh))
             break;
+		
+		rc += 0; // silent "variable not used" warning
```

Comment:
this variable used inside ```#if```
to fix need to rewrite usage of 'rc' in this function